packages/net/http/payload.pony

```diff
@@ -243,3 +242,26 @@ class iso Payload
     end
 
     list
+
+  fun string(): String =>
+    var str = proto + status.string() + method + "\n"
```

Comment:
There are a lot of string concatenation in this method. Each time strings are concatenated, a new instance is created and it's not efficient. Pony capabilities helps at this point allowing to create a mutable string and append to it. See [`printargs.pony`][1] for example.

[1]: https://github.com/ponylang/ponyc/blob/2b844f00aa9d8deb8cdd60031b78c01825ca76c3/examples/printargs/printargs.pony